eng/Version.Details.xml

```diff
@@ -172,7 +172,7 @@
     </Dependency>
     <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build" Version="6.0.0-servicing.22320.1">
```

Comment:
Should the SB and SBRP intermediate versions be updated with the hashes?  I don't think they matter for source-build but Maestro might try to update them.